src/FSharp.Azure.StorageTypeProvider.Runtime/FSharp.Azure.StorageTypeProvider.Runtime.fsproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="FSharp.Azure.StorageTypeProvider.Runtime.fs"/>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\FSharp.Azure.StorageTypeProvider.DesignTime\FSharp.Azure.StorageTypeProvider.DesignTime.fsproj" />
+  </ItemGroup>
+  <Target Name="MoveDesignTimeAssembliesToTypeProviderFolder" AfterTargets="AfterBuild">
+    <CreateItem Include="..\FSharp.Azure.StorageTypeProvider.DesignTime\bin\$(Configuration)\netstandard2.0\publish\*.dll;..\FSharp.Azure.StorageTypeProvider.DesignTime\bin\$(Configuration)\netstandard2.0\publish\*.pdb">
+      <Output TaskParameter="Include" ItemName="DesignTimeBinaries" />
+    </CreateItem>
+    <Copy SourceFiles="@(DesignTimeBinaries)" DestinationFolder="$(OutputPath)../typeproviders/fsharp41/netstandard2.0" />
```

Comment:
This one is key, a real pain in the ass, and also something I intend on shipping in the TPSDK so that we don't utilize an MSBuild hack that it currently suggests